### PR TITLE
Recommend mprven=1

### PR DESCRIPTION
**Pull request by pdonahue-ventana**
Proposed solution to the mprven concerns.  Note that this is non-normative so there's no compatibility issue.

@allenjbaum @gfavor

### CODE REVIEW

File: core_debug.tex [hunk 1]
```diff
@@ -54,9 +54,14 @@ \section{Debug Mode} \label{debugmode}
 \end{steps}
 
 \begin{commentary}
-    In general, the debugger is expected to be able to simulate all the effects of \FcsrMcontrolMprv.
-    The exception is the case of Sv32 systems, which need \FcsrMcontrolMprv functionality in order to access
-    34-bit physical addresses. Other systems are likely to tie \FcsrDcsrMprven to 0.
+    When mprven=1, the external debugger can set MPRV and MPP appropriately to
```
Comment (timsifive) [inline core_debug.tex:57]: mprven -> \FcsrDcsrMprven

File: core_debug.tex [hunk 2]
```diff
@@ -54,9 +54,14 @@ \section{Debug Mode} \label{debugmode}
...
+    have hardware perform memory accesses with the appropriate endianness, address
+    translation, permission checks, and PMP/PMA checks (subject to relaxedpriv).
```
Comment (timsifive) [inline core_debug.tex:59]: relaxedpriv -> FdmAbstractcsRelaxedpriv

File: core_debug.tex [hunk 3]
```diff
@@ -54,9 +54,14 @@ \section{Debug Mode} \label{debugmode}
...
+    This is also the only way to access all of physical memory when 34-bit physical
+    addresses are supported on a Sv32 hart. If hardware ties mprven to 0 then the
```
Comment (timsifive) [inline core_debug.tex:61]: mprven -> \FcsrDcsrMprven

File: core_debug.tex [hunk 4]
```diff
@@ -54,9 +54,14 @@ \section{Debug Mode} \label{debugmode}
...
+    external debugger is expected to simulate all the effects of MPRV, including
+    any extensions that affect memory accesses.
+    For these reasons it is recommended to tie mprven to 1.
```
Comment (timsifive) [inline core_debug.tex:64]: mprven -> \FcsrDcsrMprven

**Review by timsifive — CHANGES_REQUESTED** [review]
LGTM

File: core_debug.tex (hunk 1 above)
Comment (pdonahue-ventana) [reply core_debug.tex]: Oops.  I keep forgetting to do that.  Fixed now.

**Review by timsifive — APPROVED** [review]
Thanks.

**Pull request merged**